### PR TITLE
PIM-6333: Rename identifier to code

### PR DESCRIPTION
**Pull request by aRn0D**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: src/Pim/Component/Catalog/Model/ProductModelInterface.php [hunk 1]
```diff
@@ -32,14 +32,14 @@ public function getId(): ?int;
...
-    public function getIdentifier(): string;
+    public function getCode(): string;
 
     /**
      * Sets the product model identifier.
      *
      * @param string $identifier
      */
-    public function setIdentifier(string $identifier): void;
+    public function setCode(string $identifier): void;
```
Comment (jjanvier) [inline src/Pim/Component/Catalog/Model/ProductModelInterface.php]: $code

File: src/Pim/Component/Catalog/Model/ProductModel.php [hunk 2]
```diff
@@ -84,17 +84,17 @@ public function getId(): ?int
...
     {
-        return $this->identifier;
+        return $this->code;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function setIdentifier(string $identifier): void
+    public function setCode(string $identifier): void
```
Comment (grena) [inline src/Pim/Component/Catalog/Model/ProductModel.php]: $code

File: src/Pim/Component/Catalog/Normalizer/Standard/ProductModelNormalizer.php [hunk 3]
```diff
@@ -32,7 +32,7 @@ public function normalize($productModel, $format = null, array $context = array(
             throw new \LogicException('Serializer must be a normalizer');
         }
 
-        $data[self::FIELD_IDENTIFIER] = $productModel->getIdentifier();
+        $data[self::FIELD_IDENTIFIER] = $productModel->getCode();
```
Comment (grena) [inline src/Pim/Component/Catalog/Normalizer/Standard/ProductModelNormalizer.php]: FIELD_CODE?

**Comment by aRn0D** [comment]
Closed because of the CI...

**Pull request closed without merging**